README.md

```diff
@@ -39,10 +39,8 @@ To run this project is as simple as deploying it to a balenaCloud application; n
 By default, your device will be displayed as `balenaSound xxxx` when you search for Bluetooth devices.
 You can change this using `BLUETOOTH_DEVICE_NAME` environment variable that can be set in balena dashboard
 (navigate to dashboard -> app -> device -> device variables).
-![Setting the device name](images/device-name-config.png)
 
-If you modify the Bluetooth device name after deploying the application, you should restart `bluetooth-audio` service
-to apply the changes.
```

Comment:
@zrzka isn't this necessary?

Reply:
The change of variable would cause a restart anyway

Reply:
@roman-mazur no, because whenever you change variable / configuration / ... on the balenaCloud dashboard, supervisor is notified and automatically restarts all related services. Device variable - all services, service variable - one service, etc.

Reply:
Ah, indeed. You need to start a new container to re-set the variable.